ui/webui/src/components/HeaderKebab.jsx

```diff
@@ -143,32 +139,40 @@ export const HeaderKebab = ({ reportLinkURL, isConnected }) => {
             {_("Report Issue")}
         </DropdownItem>,
     ];
+
     return (
         <>
             <Dropdown
-              position={DropdownPosition.right}
+              isOpen={isOpen}
               onSelect={onSelect}
-              toggle={
-                  <KebabToggle
+              popperProps={{ position: "right" }}
+              position="right"
```

Comment:
I dont see any 'position' property present on the Dropdown component documentation. Did you see this somewhere?

Reply:
Yes, its there https://www.patternfly.org/components/menus/dropdown#dropdownpopperprops

Reply:
that's for the popperProps on line 148. 
I see in next line there is position property directly on the Dropdown. That's the problematic one.

Reply:
oh, i seee. Yes this is mistake .